src/Adapters/ADT3DSceneAdapter.ts

```diff
@@ -60,14 +61,23 @@ export default class ADT3DSceneAdapter {
         );
         /**
          * Check if class has been initialized with CORS enabled or if origin matches dev or prod explorer urls,
-         * override if proxy is forced by feature flag
+         * override if CORS is forced by feature flag
          *  */
         this.useProxy =
-            useProxy ||
-            validateExplorerOrigin(window.origin) ||
-            localStorage.getItem(
-                LOCAL_STORAGE_KEYS.FeatureFlags.Proxy.forceProxy
-            ) === 'true';
+            (useProxy || !validateExplorerOrigin(window.origin)) &&
+            !(
+                localStorage.getItem(
+                    LOCAL_STORAGE_KEYS.FeatureFlags.Proxy.forceCORS
+                ) === 'true'
+            );
+
+        this.useBlobProxy =
+            (useBlobProxy || !validateExplorerOrigin(window.origin)) &&
+            !(
```

Comment:
can we move the local storage read up to the module scope so we don't need to do it on every call? it's gonna be the same value for the life of the app hopefully so we can save the perf and the ease of reading.

Reply:
what do you mean module scope?

Reply:
top of da file.

Reply:
done